### ISSUE TITLE
mod_list.active_with_order(); prioritizes .zip files over unzipped mods.

### ISSUE THREAD

**Issue by notnotmelon** [fgardt/factorio-scanner#175]
I have two versions of the `achievement-enabler` mod

![image](https://github.com/user-attachments/assets/95f4437a-c05b-47b4-96ce-9518a3e5e4bd)

The expected behavior is that the .zip file is ignored and only the unzipped file is loaded, however I experience the opposite behavior.

**Comment by fgardt** [comment]
is the vanilla behaviour to favour the folder variant if 2 identical versions of the same mod exist?

**Comment by redruin1** [comment]
I don't think it's documented anywhere, but observation from my experience indicates that folders are indeed preferred if all else is equal.

I suspect this is to prefer "in-development" mods, since you would probably only compress a mod when ultimately distributing it.

**Comment by fgardt** [comment]
yeah there is a specific ordering in regards to zipped vs unzipped aswell as unzipped folders that have a version suffix or not
